project/api/models.py

```diff
@@ -7,42 +7,42 @@
 from colorfield.fields import ColorField
 
 # Create your models here.
-class Nation(models.Model):
-    """
-    Cultural/governmental entity. Serves as foreign key for most Territories
-    """
+class Entity(models.Model):
     name = models.TextField(max_length=100,
                             help_text="Canonical name, should not include any epithets, must be unique",
                             unique=True)
     url_id = models.SlugField(max_length=75,
-                              help_text="Identifier used to lookup nations in the URL, "
+                              help_text="Identifier used to lookup PoliticalEntitys in the URL, "
                                         "should be kept short and must be unique",
                               unique=True)
-    color = ColorField(help_text="Color to display on map",
-                       unique=True,
-                       null=True,
-                       blank=True)
-    history = HistoricalRecords()
-
-    ## Flavor fields
-
-    # required fields
     references = ArrayField(
```

Comment:
Consider adding `help_text` to the `references` and `links` fields.

Reply:
Is this still necessary if we don't plan to use the admin page or django templates? Might remove all the help texts

Reply:
It's still helpful for people reading the code in the future. This is my bias speaking though, as my personal preference is to tend towards the side of overdocumentation. Up to you if you think the documentation on the frontend is enough.

Reply:
Fair enough, I'll add this.